prisma/seedTeamMembers.ts

```diff
@@ -0,0 +1,25 @@
+import { PrismaClient, Role } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+// This function will create 10 team members for testing the list of team members
```

Comment:
Make sure you have 80 characters max on each line, also let's use some comments above each `for` loop